.github/workflow/worker-push.yml

```diff
@@ -0,0 +1,37 @@
+name: Containerize worker
+
+on:
```

Comment:
Can we add a filter here to ensure this only deploys if the `worker` package has changed?

Reply:
@hkirat  done